packages/dds/tree/src/feature-libraries/treeCompressionUtils.ts

```diff
@@ -21,3 +21,23 @@ export enum TreeCompressionStrategy {
 	 */
 	Uncompressed = 1,
 }
+
+/**
+ * This is an internal type that extends TreeCompressionStrategy.
+ * It is used to add compression strategies that are not intended for public use just yet.
+ * @internal
+ */
+export enum TreeCompressionStrategyExtended {
+	/**
+	 * Optimized for encoded size, same as TreeCompressionStrategy.Compressed. It also enables incremental encoding
+	 * of the data.
+	 */
+	CompressedIncremental = 2,
+}
+
+/**
+ * @internal
```

Comment:
These are not package exported, so should not be marked as `@internal`

This should have more documentation that just an API tag.

As this is not package exported, and thus not "internal" from a release level tag perspective, calling it TreeCompressionStrategyPrivate might be better (ITreePrivate is an example of another type we have using that pattern: ITreeInternal is package exported as internal and ITreePrivate has package private extras)